contrib/redwood/plugins.yml

```diff
@@ -24,3 +24,11 @@
   description: |
     Tutor plugin to install Meilisearch and enable course content search in Studio.
     See the README on GitHub for further setup instructions.
+
+- name: k8s_harmony
+  src: -e 'git+https://github.com/openedx/openedx-k8s-harmony.git#egg=tutor-contrib-harmony-plugin&subdirectory=tutor-contrib-harmony-plugin'
+  url: https://github.com/openedx/openedx-k8s-harmony
+  author: eduNEXT, OpenCraft, and contributors
```

Comment:
Similar to the Aspects plugin (see my [comment](https://github.com/overhangio/tpi/pull/4#discussion_r1834151846)) we will need an explicit email address, such that both users and TPI maintainers know who to reach out to in case of issue.

Reply:
You can put `help@opencraft.com` as an email if strictly required, but we'd strongly prefer users seek support via GitHub Issues on this project, or via Open edX Discourse or Slack.